### PR TITLE
feat(geolocation): geolocation utility

### PR DESCRIPTION
**Pull request by annawen1**
### Related Ticket(s)

[Locale Selector: develop a utility to fetch the user's geo-ip #1816](https://zenhub.ibm.com/app/workspaces/digital-design-system-5b3a815a8bf6a932efa6fd03/issues/webstandards/digital-design/1816)

### Description

Create utility that checks geolocation of user to derive country code and uses browser language preference to pull language code.

### Changelog

**New**

- mock axios.get
- add utility for geolocation and unit test
- set up axios/rollup configuration

### CODE REVIEW

File: packages/utilities/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+GEO_API=https://api.www.s81c.com/webmaster/dbip/
```
Comment (jeffchew) [inline packages/utilities/.env]: @annawen1 this file shouldn't be checked in, was `.env` added to `.gitignore`?

File: packages/utilities/src/utilities/geolocation/geolocation.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      headers: {
+        'Content-Type': 'application/json; charset=utf-8',
+      },
+    })
+    .then(response => response.data);
+
+  const cc = location && location.country;
+
+  // get language preference from browser
+  const lc = window.navigator.language;
```
Comment (jeffchew) [inline packages/utilities/src/utilities/geolocation/geolocation.js]: @annawen1 use `window-or-global` here

File: packages/utilities/.env (hunk 1 above)
Comment (annawen1) [reply packages/utilities/.env]: whoops!

File: packages/utilities/src/utilities/geolocation/geolocation.js (hunk 2 above)
Comment (annawen1) [reply packages/utilities/src/utilities/geolocation/geolocation.js]: done

**Pull request merged**